pylintrc

```diff
@@ -426,6 +426,8 @@ good-names=i,
            k,
            ex,
            Run,
+           to,
```

Comment:
I think we should seriously consider email_to and email_cc instead. This way we would not have to make exclusions here. I think exclusions in pylintrc should be "generic" - i.e. names that are universally used as short names in well-known way (indexes, exceptions etc.). "to" and "cc" are very specific to email case.

Reply:
Good idea. I was thinking on how to replace `to` and `cc` and had no ideas, but `email_to` and `email_cc` do look good, thanks.

Reply:
In the end I opted for `to_addr`, `cc_addr` and `bcc_addr` (the last one for consistency).
e68d601b5acd9b1773c218219eeb7fb33e068e6c